<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.38.1&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 35 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_30.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,611,384</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">63.9%
</td><td align="center">1.9
/&nbsp;90.2
</td><td align="center">63</td><td align="center"><font class="snp_type_nonsynonymous">V152L</font>&nbsp;(<font class="mutation_in_codon">G</font>TC&rarr;<font class="mutation_in_codon">C</font>TC)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS07530</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_33.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,939,477</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">45.5%
</td><td align="center">5.0
/&nbsp;44.7
</td><td align="center">22</td><td align="center">pseudogene&nbsp;(6/369&nbsp;nt)</td><td align="center"><i>antC</i></td><td align="left">anthranilate dioxygenase reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_36.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,396</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">44.4%
</td><td align="center">60.3
/&nbsp;35.3
</td><td align="center">53</td><td align="center">intergenic&nbsp;(&#8209;8/&#8209;251)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_35.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,395</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">42.9%
</td><td align="center">71.1
/&nbsp;40.1
</td><td align="center">56</td><td align="center">intergenic&nbsp;(&#8209;7/&#8209;252)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_37.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,397</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">40.0%
</td><td align="center">70.6
/&nbsp;30.5
</td><td align="center">53</td><td align="center">intergenic&nbsp;(&#8209;9/&#8209;250)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_42.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,736,592</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">39.7%
</td><td align="center">130.9
/&nbsp;55.8
</td><td align="center">80</td><td align="center"><font class="snp_type_nonsynonymous">S479T</font>&nbsp;(A<font class="mutation_in_codon">G</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>fadH</i></td><td align="left">NADPH&#8209;dependent 2,4&#8209;dienoyl&#8209;CoA reductase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_46.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,125,709</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">37.6%
</td><td align="center">173.6
/&nbsp;77.3
</td><td align="center">106</td><td align="center"><font class="snp_type_nonsynonymous">G330R</font>&nbsp;(<font class="mutation_in_codon">G</font>GC&rarr;<font class="mutation_in_codon">C</font>GC)&nbsp;</td><td align="center"><i>ACX60_RS14780</i></td><td align="left">acyl&#8209;CoA dehydrogenase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_31.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,611,385</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">36.8%
</td><td align="center">82.6
/&nbsp;42.4
</td><td align="center">68</td><td align="center"><font class="snp_type_nonsynonymous">V152A</font>&nbsp;(G<font class="mutation_in_codon">T</font>C&rarr;G<font class="mutation_in_codon">C</font>C)&nbsp;&Dagger;</td><td align="center"><i>ACX60_RS07530</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_39.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,668,823</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">36.2%
</td><td align="center">109.1
/&nbsp;22.6
</td><td align="center">80</td><td align="center"><font class="snp_type_nonsynonymous">L216F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS12775</i></td><td align="left">GntR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_34.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,204,394</td><td align="right">0</td><td align="center">A</td><td align="center">G</td><td align="right">35.7%
</td><td align="center">71.0
/&nbsp;31.3
</td><td align="center">56</td><td align="center">intergenic&nbsp;(&#8209;6/&#8209;253)</td><td align="center"><i>ACX60_RS10510/ACX60_RS10515</i></td><td align="left">hypothetical protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_48.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,632,436</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">35.0%
</td><td align="center">221.4
/&nbsp;53.9
</td><td align="center">143</td><td align="center"><font class="snp_type_nonsynonymous">L70F</font>&nbsp;(TT<font class="mutation_in_codon">A</font>&rarr;TT<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS17130</i></td><td align="left">MFS transporter</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_41.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,684,073</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">31.8%
</td><td align="center">183.0
/&nbsp;10.6
</td><td align="center">90</td><td align="center"><font class="snp_type_nonsynonymous">I242M</font>&nbsp;(AT<font class="mutation_in_codon">T</font>&rarr;AT<font class="mutation_in_codon">G</font>)&nbsp;</td><td align="center"><i>ACX60_RS12840</i></td><td align="left">5&#8209;dehydro&#8209;4&#8209;deoxyglucarate dehydratase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_40.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,684,049</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">30.0%
</td><td align="center">198.2
/&nbsp;11.2
</td><td align="center">93</td><td align="center"><font class="snp_type_nonsynonymous">I250M</font>&nbsp;(AT<font class="mutation_in_codon">T</font>&rarr;AT<font class="mutation_in_codon">G</font>)&nbsp;</td><td align="center"><i>ACX60_RS12840</i></td><td align="left">5&#8209;dehydro&#8209;4&#8209;deoxyglucarate dehydratase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_10.html">*</a></td><td align="center">CP012005</td><td align="right">126,929</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">29.5%
</td><td align="center">61.3
/&nbsp;18.5
</td><td align="center">44</td><td align="center"><font class="snp_type_nonsynonymous">I167T</font>&nbsp;(A<font class="mutation_in_codon">T</font>T&rarr;A<font class="mutation_in_codon">C</font>T)&nbsp;</td><td align="center"><i>ACX60_18905</i></td><td align="left">TetR family transcriptional regulator</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_50.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">3,742,058</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">29.0%
</td><td align="center">299.4
/&nbsp;21.0
</td><td align="center">107</td><td align="center"><font class="snp_type_nonsense">L375*</font>&nbsp;(T<font class="mutation_in_codon">T</font>A&rarr;T<font class="mutation_in_codon">G</font>A)&nbsp;</td><td align="center"><i>ACX60_RS17660</i></td><td align="left">type VI secretion system protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_27.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">1,120,875</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">28.8%
</td><td align="center">96.7
/&nbsp;12.4
</td><td align="center">61</td><td align="center">intergenic&nbsp;(+40/+16)</td><td align="center"><i>ACX60_RS05265/ACX60_RS05270</i></td><td align="left">RNA polymerase&#8209;associated protein RapA/membrane protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_21.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">682,086</td><td align="right">0</td><td align="center">A</td><td align="center">C</td><td align="right">28.4%
</td><td align="center">136.8
/&nbsp;14.9
</td><td align="center">68</td><td align="center">intergenic&nbsp;(+50/+75)</td><td align="center"><i>ACX60_RS03295/ACX60_RS03300</i></td><td align="left">membrane protein/hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_20.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">607,375</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">27.6%
</td><td align="center">209.9
/&nbsp;32.0
</td><td align="center">116</td><td align="center"><font class="snp_type_synonymous">D178D</font>&nbsp;(GA<font class="mutation_in_codon">T</font>&rarr;GA<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>ACX60_RS02965</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_45.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,984,858</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">25.9%
</td><td align="center">293.5
/&nbsp;66.5
</td><td align="center">127</td><td align="center">noncoding&nbsp;(344/355&nbsp;nt)</td><td align="center"><i>ACX60_RS19000</i></td><td align="left">bacterial RNase P</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_38.html">*</a></td><td align="center">NZ_CP012004</td><td align="right">2,624,871</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">25.3%
</td><td align="center">230.0
/&nbsp;25.5
</td><td align="center">89</td><td align="center"><font class="snp_type_nonsynonymous">N91H</font>&nbsp;(<font class="mutation_in_codon">A</font>AC&rarr;<font class="mutation_in_codon">C</font>AC)&nbsp;</td><td align="center"><i>ACX60_RS12505</i></td><td align="left">hypothetical protein</td></tr>
</table>

<p>
<table border="0" cellspacing="1" cellpadding="3">
<tr><th colspan="12" align="left" class="new_junction_header_row">Marginal new junction evidence (sorted from low to high skew)</th></tr>
<!-- Header Lines for New Junction -->
<tr>
<th colspan="2">&nbsp;</th>
<th>seq&nbsp;id</th>
<th>position</th>
<th>reads&nbsp;(cov)</th>
<th>reads&nbsp;(cov)</th>
<th>score</th>
<th>skew</th>
<th>freq</th>
<th>annotation</th>
<th>gene</th>
<th width="100%">product</th>
</tr>

<!-- Item Lines for New Junction -->
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_62.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_62.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;418620</td><td align="center" class="junction_gene">146 (1.330)</td><td rowspan="2" align="center">5 (0.050)</td>
<td rowspan="2" align="center">5/256</td>
<td rowspan="2" align="center">4.6</td>
<td rowspan="2" align="center">4.1%</td>
<td align="center" class="junction_gene">coding&nbsp;(1193/1848&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS01970</i></td>
<td class="junction_gene">DEAD/DEAH box helicase</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_62.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">418840&nbsp;=</td>
<td align="center" class="junction_gene">104 (1.080)</td><td align="center" class="junction_gene">noncoding&nbsp;(60/240&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>repeat_region</i></td>
<td class="junction_gene">repeat region</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_64.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_64.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;2341863</td><td align="center" class="junction_gene">47 (0.430)</td><td rowspan="2" align="center">4 (0.040)</td>
<td rowspan="2" align="center">3/246</td>
<td rowspan="2" align="center">5.3</td>
<td rowspan="2" align="center">6.6%</td>
<td align="center" class="junction_gene">coding&nbsp;(859/1236&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS11185</i></td>
<td class="junction_gene">stress&#8209;induced protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_64.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">2342449&nbsp;=</td>
<td align="center" class="junction_gene">74 (0.800)</td><td align="center" class="junction_gene">coding&nbsp;(273/1236&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS11185</i></td>
<td class="junction_gene">stress&#8209;induced protein</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="2"><a href="evidence/JC_63.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_63.html">?</a></td>
<td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">=&nbsp;418620</td><td align="center" class="junction_gene">146 (1.330)</td><td rowspan="2" align="center">4 (0.040)</td>
<td rowspan="2" align="center">3/258</td>
<td rowspan="2" align="center">5.5</td>
<td rowspan="2" align="center">3.6%</td>
<td align="center" class="junction_gene">coding&nbsp;(1193/1848&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>ACX60_RS01970</i></td>
<td class="junction_gene">DEAD/DEAH box helicase</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_0">
<td rowspan="1"><a href="evidence/JC_SIDE_2_63.html">?</a></td><td rowspan="1" class="junction_gene">NZ_CP012004</td>
<td align="center" class="junction_gene">418927&nbsp;=</td>
<td align="center" class="junction_gene">86 (0.890)</td><td align="center" class="junction_gene">noncoding&nbsp;(147/240&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>repeat_region</i></td>
<td class="junction_gene">repeat region</td>
</tr>
<!-- Side 1 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="2"><a href="evidence/JC_58.html">*</a></td>
<td rowspan="1"><a href="evidence/JC_SIDE_1_58.html">?</a></td>
<td rowspan="1" class="junction_gene">CP000523</td>
<td align="center" class="junction_gene">=&nbsp;3662</td><td align="center" class="junction_gene">871 (1.230)</td><td rowspan="2" align="center">7 (0.010)</td>
<td rowspan="2" align="center">7/266</td>
<td rowspan="2" align="center">21.0</td>
<td rowspan="2" align="center">0.94%</td>
<td align="center" class="junction_gene">coding&nbsp;(615/1215&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>A1S_3473</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
<!-- Side 2 Item Lines for New Junction -->
<tr class="mutation_table_row_1">
<td rowspan="1"><a href="evidence/JC_SIDE_2_58.html">?</a></td><td rowspan="1" class="junction_gene">CP000523</td>
<td align="center" class="junction_gene">4420&nbsp;=</td>
<td align="center" class="junction_gene">679 (1.050)</td><td align="center" class="junction_gene">coding&nbsp;(64/777&nbsp;nt)</td>
<td align="center" class="junction_gene"><i>A1S_3474</i></td>
<td class="junction_gene">hypothetical protein</td>
</tr>
</table>
</body></html>